helm/prometheus-rules/templates/alerting-rules/fluxcd.rules.yml

```diff
@@ -0,0 +1,63 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: fluxcd.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: fluxcd
+    rules:
+    - alert: FluxCDHelmReleaseFailed
+      annotations:
+        description: |-
+          {{`FluxCD HelmRelease on {{ $labels.installation }}/{{ $labels.cluster_id }} in {{ $labels.namespace }}/{{ $labels.name }} is stuck in Failed state.`}}
+        opsrecipe: fluxcd/failing
+      expr: gotk_reconcile_condition{type="Ready", status="False", kind="HelmRelease"}
+      for: 10m
+      labels:
+        area: kaas
```

Comment:
Is this actually kaas?

Reply:
I based the area on ArgoCD alerts. Does `managedservices` better?